### DISCUSSION TITLE
Accent Color not work in 3.4.1 Version

### DISCUSSION THREAD

**Discussion by ashkankiani** [tailwindlabs/tailwindcss#12937] [Help] [answered]
**What version of Tailwind CSS are you using?**
3.4.1

**What build tool (or framework if it abstracts the build tool) are you using?**
  "devDependencies": {
    "@tailwindcss/forms": "^0.5.7",
    "@tailwindcss/typography": "^0.5.10",
    "autoprefixer": "^10.4.17",
    "eslint": "^8",
    "eslint-config-next": "14.1.0",
    "eslint-config-prettier": "^9.1.0",
    "postcss": "^8.4.35",
    "tailwindcss": "^3.4.1"
  }

**What version of Node.js are you using?**
V20.8.0

**What browser are you using?**

All Browser Chrome, Mozila and Edge

**What operating system are you using?**

Windows 11

**Reproduction URL**
localhost 

**Describe your issue**

use Tailwind in nextjs and use code:
      <label>
        <input type="checkbox" className="accent-red-500" checked /> Customized
      </label>
      <label>
        <input type="checkbox" className="accent-pink-500" checked /> Customized
      </label>
but not working, Other Tailwind versions are correct.

**Comment by ashkankiani** [comment]
code is :
`      <label>
        <input type="checkbox" className="accent-red-500" checked /> Customized
      </label>
      <label>
        <input type="checkbox" className="accent-pink-500" checked /> Customized
      </label>
`

**Comment by Vaibhavkulshrestha12** [comment]
> code is : ` <label> <input type="checkbox" className="accent-red-500" checked /> Customized </label> <label> <input type="checkbox" className="accent-pink-500" checked /> Customized </label>`

have you tried to install the accent color plugin  

plugins: [
  require('tailwind-accent-color')(),
]

try this using the yarn installation of accent color module

**Comment by ashkankiani** [comment]
There is no mention of the following library in the link below
`https://tailwindcss.com/docs/accent-color`
But again, I did this, but unfortunately it is not added.

**Comment by thecrypticace** [comment]
@ashkankiani Can you provide a reproduction?

Accent color is working correctly for me: https://play.tailwindcss.com/r5Ylcab884

on macOS:
<img width="179" alt="Screenshot 2024-02-13 at 09 21 31" src="https://github.com/tailwindlabs/tailwindcss/assets/614993/3ed66ab8-f1e8-41ec-8514-adf7479d747b">

on Windows (Edge and Chrome both work):
<img width="179" alt="Screenshot 2024-02-13 at 09 21 31" src="https://github.com/tailwindlabs/tailwindcss/assets/614993/7d39002f-8a2f-4a87-93f8-0a70069b7d5f">

I will note that if you have a high contrast theme applied in Windows like Aquatic, Desert, Dusk, or   Night Sky those will override the `accent-color` CSS property and it no longer functions but that is an intentional browser / OS behavior.

**Comment by Jonath-z** [comment]
@ashkankiani  are you using any UI library?

**Comment by ashkankiani** [comment]
No, I don't use it and my problem was not solved in NEXTJS. There is no problem in simple mode.

**Reply by xt0rted** [reply, marked as answer]
I just ran into this issue and found that when you're using `@tailwindcss/forms` the checked state is replaced with an svg that's hard coded to white. The accent color can be set using the text color classes like `text-red-500`.

Here's a modified example from https://github.com/tailwindlabs/tailwindcss/discussions/12937#discussioncomment-8454436

https://play.tailwindcss.com/YikF8AkOS3

**Comment by ashkankiani** [comment]
thank you. solved for me.
add class text-yellow-400 focus:ring-yellow-400
But this is not the standard method. You should use the Accent Color method of tailwinds, which interferes with the form
